modules/postgres/variables.tf

```diff
@@ -1,39 +1,70 @@
+# Removing - setting directly as part of module, leaving commented out until
+#  migration complete to make it clear that this is something we are
+#  deliberately removing.
+#variable SecurityGroups {
+#  type        = "commadelimitedlist"
+#  description = "A list of security group ids, such as sg-a123fd85"
+#}
+
 variable SubnetID {
-  type        = "list"
+  type        = "string"
```

Comment:
I've been removing explicit `type = "string"` s as it's the default. Again, not a blocker just a question.

Reply:
I like the clarity, but only mildly.  No real pref, so good to drop if you'd like.